meowth/__main__.py

```
@@ -1433,7 +1433,7 @@ def cleanup_code(content):
     response = ""
     region_info_dict = guild_dict[guild.id]['configure_dict']['regions']['info']
     enabled_roles = set([r.get('role', None) for r in region_info_dict.values()])
-    requested_roles = set([r for r in re.split(r'\s*,\s*', region_names.lower().strip()) if r])
+    requested_roles = set([r for r in re.split(r'\s*,\s*', region_names.lower().replace(" ", "")) if r])
```

this kind of formatting is problematic because it may not conform to region names a server owner actually wants to set up. I was going to suggest a universal region name normalization to be applied here and elsewhere, but I think the root cause issue you're trying to address is people sending a region command that partially succeeds or fails silently due to whitespace (assuming something like "des moines" vs "desmoines")

the best fix instead would be to change the feedback mechanism for the region commands from just giving a checkmark or nothing to a success/error list like the subscription command does, and adding the list of available to join/leave roles in that message too.

---

Yes the feedback is there, but the bot cleans it up after a relatively short time.

Overall it's likely the least of the pain points people have with regions and we could certainly leave it as is.

---

I doubled the feedback message duration. Alternatively it could stay up indefinitely. That part is flexible.

---

yes, let's leave these as-is then and revert the `replace()` calls back to `strip()`

---

@tehstone I keep checking in on this PR but these changes still need to be reverted